src/emailpro/external-contact/update/emailpro-external-contact-update.controller.js

```diff
@@ -0,0 +1,94 @@
+angular
+    .module("Module.emailpro.controllers")
+    .controller("EmailProExternalContactsModifyCtrl", class EmailProExternalContactsModifyCtrl {
+        constructor ($scope, $stateParams, EmailPro, EmailProExternalContacts, Alerter, translator) {
+            this.$scope = $scope;
+            this.$stateParams = $stateParams;
+            this.EmailPro = EmailPro;
+            this.EmailProExternalContacts = EmailProExternalContacts;
+            this.Alerter = Alerter;
+            this.translator = translator;
+        }
+
+        $onInit () {
+            this.previousModel = angular.copy(this.$scope.currentActionData);
+            this.model = angular.copy(this.$scope.currentActionData);
+
+            this.isLoading = false;
+            this.emailValidationRegex = "^[a-zA-Z0-9]+([\.+\-\w][a-zA-Z0-9]+)*@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$";
```

Comment:
You shoud use a generic method for this check like validator.js ;-)

Reply:
Validator.js validates email addresses the way they should be validated, no the way OVH accepts them. I changed the Validator service to include an email address validation function.